### ISSUE TITLE
CID Checker reports duplicate data

### ISSUE THREAD

**Issue by Lind111** [fidlabs/allocator-tooling#73]
![image](https://github.com/user-attachments/assets/46112014-d73d-4097-8e3f-6bdb71a31308)
![image](https://github.com/user-attachments/assets/78f75d15-f522-40d2-9a65-d96487b0fa6f)
These two SPs have not been sealed for almost 10 month
Inconsistency with on-chain data obtained by [third parties](https://ehume.com/#/clientDetail?clientAddress=f1txfwjexmp4gp3dqdglgc3i7qacnyseocs4cecky&dataCapConsumeStr=3.53%20PB&dataCapReleasedStr=3.46%20PB&dataCapTotalStr=3.46%20PB&dataCapNotConsumeStr=-64.75%20TB)
![image](https://github.com/user-attachments/assets/3ca86d62-8155-4625-a322-df62f47f2545)

Problem report location
https://github.com/Lind111/EF/issues/9#issuecomment-2357653616

This Application has been affected by the events of STFIL's
Please assist in troubleshooting the problem, which affects the decision-making of the allocators.

**Comment by kacperzuk-neti** [comment]
I confirm that the report is incorrect (some data for these providers is duplicated). We're working with the data source team on amending this.

**Comment by Lind111** [comment]
hey @kacperzuk-neti 

I'd like to know how long this will take to process

**Comment by kacperzuk-neti** [comment]
Unclear, we're still working on how to handle this.

**Comment by Lind111** [comment]
@kacperzuk-neti  Hi  
What's the latest news here?This issue is already affecting my review

**Comment by kacperzuk-neti** [comment]
@Lind111 team responsible for the data source is working on a fix. It should be ready today, so we'll update reporting tools to support new changes tomorrow an then reports should be correct. I'll update this ticket when that's done.

**Comment by kacperzuk-neti** [comment]
@Lind111 fix in the data source is taking longer than expected, fix is delayed.

**Comment by hash889900** [comment]
https://github.com/hash889900/HashTeam/issues/3
https://github.com/hash889900/HashTeam/issues/8

Looks like it's the same fault.

**Comment by ssnks1** [comment]
Please help us correct it, thank you

**Comment by kacperzuk-neti** [comment]
Some mitigations that reduce the amount of reported duplicated data were already implemented, but we're still waiting on a complete fix.

**Comment by kacperzuk-neti** [comment]
Just to update all watching this issue - the root cause was identified, but fixing this is non-trivial. Our data provider is actively working on this all this time. DDO should be unaffected by this issue.

**Comment by TOPPOOL-LEE** [comment]
https://github.com/TOPPOOL-LEE/Allocator-Pathway-TOP-POOL/issues/40
<img width="776" alt="WX20241104-171608@2x" src="https://github.com/user-attachments/assets/fea73ee6-a0e5-4459-b5c8-98afb973bd6e">

**Comment by udi1231239** [comment]
https://github.com/TOPPOOL-LEE/Allocator-Pathway-TOP-POOL/issues/40Dear technicians, could you help us  to solve the problem? Thank you. @kacperzuk-neti

**Comment by kacperzuk-neti** [comment]
@udi1231239 our data provider is working on this problem with the highest priority.

**Comment by TOPPOOL-LEE** [comment]
It's been fixed, thank you @kacperzuk-neti

**Comment by kacperzuk-neti** [comment]
Our data provider reports that the issue should be fixed and indeed I'm unable to find any problematic cases.
@Lind111 @hash889900 @ssnks1 @udi1231239 please let me know if you still see any issues with reports generated since today.

**Comment by TOPPOOL-LEE** [comment]
@kacperzuk-neti But our closed LDN has not been fixed.

https://github.com/TOPPOOL-LEE/Allocator-Pathway-TOP-POOL/issues/34
https://github.com/TOPPOOL-LEE/Allocator-Pathway-TOP-POOL/issues/20
https://github.com/TOPPOOL-LEE/Allocator-Pathway-TOP-POOL/issues/22

**Comment by kacperzuk-neti** [comment]
@TOPPOOL-LEE thanks for the report, we will investigate.

**Comment by kacperzuk-neti** [comment]
@TOPPOOL-LEE please open a new issue, describing in detail what is the problem. We couldn't find anything wrong with the linked reports.

https://github.com/TOPPOOL-LEE/Allocator-Pathway-TOP-POOL/issues/20 and https://github.com/TOPPOOL-LEE/Allocator-Pathway-TOP-POOL/issues/22 don't report excessive duplicate data.
https://github.com/TOPPOOL-LEE/Allocator-Pathway-TOP-POOL/issues/34 reports duplicate data for a single SP, but it's a real duplicate data sealed by the SP, not a reporting issue.
